app/src/main/java/com/pwhs/quickmem/presentation/app/classes/remove_members/RemoveMembersScreen.kt

```
@@ -0,0 +1,21 @@
+@Composable
+fun RemoveMemberScreen(
+    modifier: Modifier = Modifier,
+    navigator: DestinationsNavigator
+) {
+    RemoveMember()
+}
+
+@Composable
+fun RemoveMember() {
+
+}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)